### PR TITLE
[Akamai] Formatting changes, check dashboards, better error tagging ingest pipeline

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

This is part of a bigger cleanup that relates to https://github.com/elastic/security-team/issues/5910.

The tasks, when applicable was:

- [x] Formatting with https://github.com/efd6/fumpt
- [x] Add dynamic ECS template
- [x] Update format_version to 2.3.0+
- [x] Check if any ignore_failures can be removed
- [x] Update on_failure to include proper event.kind.
- [x] Update on_failure error message to be more descriptive
- [x] If system/pipeline tests are missing, add them when possible.
- [x] If httpjson, does it have drop processor for empty events?
- [ ] If no dashboards, add them.
- [x] If dashboards exist, replace with lens when needed, update dashboards if applicable.

The PR looks like it is changing more than it is, because most of them are formatting changes.

Some minor tweaks to the dashboard has been added:
1. Fixed some vertical line descriptions for some of the graphs.
2. Added table of contents.
3. Fixed some panel titles
4. Changed layout a bit.
5. All is already lens, so no need to convert any of them.
image

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #https://github.com/elastic/security-team/issues/5910

### CODE REVIEW

File: packages/akamai/changelog.yml [hunk 1]
```diff
@@ -1,85 +1,90 @@
 # newer versions go on top
-- version: "2.4.0"
+- version: 2.5.0
+  changes:
+    - description: Updating dashboards and enabling ECS dynamic templates
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/4576
```
Comment (bhapas) [inline packages/akamai/changelog.yml:6]: Probably need to update PR number here

File: packages/akamai/data_stream/siem/_dev/test/system/test-default-config.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    client_secret: qazwsx
+    ssl.verification_mode: none
     access_token: abcd
+    api_host: http://{{Hostname}}:{{Port}}
+    client_secret: qazwsx
+    client_token: qwerasdf
     config_ids: aaaa
-    ssl.verification_mode: none
+    preserve_original_event: true
+vars: null
```
Comment (kcreddy) [inline packages/akamai/data_stream/siem/_dev/test/system/test-default-config.yml:12]: Do we need to explicitly specify it as `null` or instead ignore setting it altogether?

File: packages/akamai/data_stream/siem/fields/fields.yml [hunk 3]
```diff
@@ -1,120 +1,118 @@
...
         Indicators that increased the calculated risk score. For example, the value udfp represents the risk of the device fingerprint based on the user's behavioral profile.
 
+    - name: user_risk.score
+      type: long
+      description: >
+        Calculated risk scores. Scores range from 0 (no risk) to 100 (the highest possible risk).
+
+    - name: user_risk.status
+      type: long
+      description: "Status code indicating any errors that might have occurred when calculating the risk score. \n"
```
Comment (kcreddy) [inline packages/akamai/data_stream/siem/fields/fields.yml:108]:
```suggestion
      description: "Status code indicating any errors that might have occurred when calculating the risk score."
```

**Review by kcreddy — APPROVED** [review]
LGTM after resolving merge conflicts. Just minor clarifications

**Pull request closed without merging**